tests/common/snappi/common_helpers.py

```diff
@@ -385,16 +386,25 @@ def get_wred_profiles(host_ans):
             }
         }
     """
-    config_facts = host_ans.config_facts(host=host_ans.hostname,
-                                         source="running")['ansible_facts']
+    if asic_value == "None":
+        config_facts = host_ans.config_facts(
+                                             host=host_ans.hostname,
+                                             source="running"
+                                             )['ansible_facts']
+    else:
+        config_facts = host_ans.config_facts(
+                                            host=host_ans.hostname,
+                                            source="running",
+                                            namespace=asic_value
+                                            )['ansible_facts']
 
     if "WRED_PROFILE" in list(config_facts.keys()):
         return config_facts['WRED_PROFILE']
     else:
         return None
 
 
-def config_wred(host_ans, kmin, kmax, pmax, profile=None):
+def config_wred(host_ans, kmin, kmax, pmax, profile=None, asic_value='None'):
```

Comment:
One basic question. Do these changes to snappi/common_helpers.py have to be backported to ixia/common_helpers.py?

Reply:
> One basic question. Do these changes to snappi/common_helpers.py have to be backported to ixia/common_helpers.py?

It seems to me that we will only use snappi/ rather than ixia/ for ixia-based RDMA tests. So we probably do not need backport this function to ixia/common_helpers.py.

Reply:
We should set the default value of asic_value to None rather than 'None'

Reply:
this has been resolved